pkg/kudoctl/cmd/test.go

```diff
@@ -0,0 +1,114 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"reflect"
+	"testing"
+
+	kudo "github.com/kudobuilder/kudo/pkg/apis/kudo/v1alpha1"
+	"github.com/kudobuilder/kudo/pkg/test"
+	testutils "github.com/kudobuilder/kudo/pkg/test/utils"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	testExample = `
+      Run tests configured by kudo-test.yaml:
+            kubectl kudo test
+
+      Load a specific test configuration:
+            kubectl kudo test --config test.yaml
+
+      Run tests against an existing Kubernetes cluster:
```

Comment:
this is pretty dangerous :) I would want to have that disabled when I have kudo CLI attached to my prod cluster...

Reply:
You have to provide a configuration and tests. I'm not sure what you're suggesting.